ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -71,16 +71,15 @@ export const appRoutes: Routes = [
                 component: DefaultLayoutComponent,
                 children: [
                     {path: '', redirectTo: "/project-list", pathMatch: "full"},
+                    {path: 'about', component: AboutPageComponent},
                     {
                         path: "configuration",
-                        resolve: { configuration: ConfigurationResolve },
```

Comment:
:+1: to that. I agree on running some autoformatter on everything after summit. But we will do it in separate PR which will take care only of formatting.

For me it is not matter of single line, but of principle. @OndraZizka You tend to have many unrelated changes in your PRs and quite often also formatting ones.